ffcx/codegeneration/expression_generator.py

```diff
@@ -64,14 +64,19 @@ def generate_geometry_tables(self):
         # Currently we only support circumradius
         ufl_geometry = {
             ufl.geometry.ReferenceCellVolume: "reference_cell_volume",
+            ufl.geometry.ReferenceNormal: "reference_facet_normals",
         }
+        # Types that are only supported for facet expressions
```

Comment:
What is this comment related to?

Reply:
Leftover from a previous iteration of this code.